relay-server/src/actors/envelopes.rs

```diff
@@ -353,6 +362,7 @@ impl fmt::Debug for SendMetrics {
             .field("buckets", &self.buckets)
```

Comment:
@jan-auer any idea why we don't use `derive(Debug)` here?

Reply:
I think we couldn't because `SendMetrics` used to hold an `Addr<Project>`. Now that `Project` is not an actor anymore, and this struct only holds a `ProjectKey`, I think we can use `derive(Debug)`.